packages/replayio/src/utils/installation/types.ts

```diff
@@ -16,9 +16,14 @@ export type Release = {
   version: string | null;
 };
 
-export type MetadataJSON = {
-  chromium: {
-    buildId: string;
-    installDate: string;
-  };
-};
+export type MetadataJSON = Partial<
+  Record<
+    Runtime,
+    | {
+        buildId: string;
+        installDate: string;
+        nativeVersion: string | null;
+      }
+    | undefined
+  >
+>;
```

Comment:
This one could help a little bit with nesting but ofc both are fine:
```suggestion
export type MetadataJSON = {
  [K in Runtime]?: {
    buildId: string;
    installDate: string;
    nativeVersion: string | null;
  }
}
```